.github/workflows/ci_release.yml

```diff
@@ -0,0 +1,95 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    # Nightly schedule pending full e2e testing with other repos
+    #schedule:
+    # Run daily at 1:15am
+    #- cron: "15 1 * * *"
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: "Semver type of new version (major / minor / patch)"
+        # Input has to be provided for the workflow to run
+        required: true
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+
+jobs:
+  hadolint:
+    uses: SkynetLabs/.github/.github/workflows/reusable_dockerfile_lint.yml@master
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.18"
+      - name: Install analyze
+        run: go install gitlab.com/NebulousLabs/analyze@latest
+      - name: Install golangci-lint
+        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.45.0
+      - name: Lint
+        run: make lint
+      - name: Run short tests
+        run: make test
+      # TODO: enable, currently tests are broken and time out
```

Comment:
all of the abuse related repos (and others actually) need mongo, so there's a make command called `start-mongo` that you'll probably need to run before running the tests. Is it possible that was the issue the tests appeared broken and time out

Reply:
We start and stop mongo when running the `test-int` tests. Maybe we should have that one here?
That being said, when I tried to do that in a GitHub action I couldn't connect to the mongo container.